### ISSUE TITLE
Install from a pure git path

### ISSUE THREAD

**Issue by xogeny** [xogeny/impact#21]
Instead of doing:

`impact.py install LibName`

It would be good to also allow a syntax like this:

`impact.py install git://github.com/LibName#X.Y.Z`

I think PIP supports a syntax like this.

**Comment by xogeny** [comment]
I thought about this and I wonder if it would make sense to check if the current directory was already a git repo and, if so, insert the dependency as a submodule?

**Comment by xogeny** [comment]
Isn't this the same as #54?  We really need some kind of clear specification of how impact should function in conjunction with version control.  These are just "one off" feature requests...but I don't have a clear image of the "big picture".

**Comment by dietmarw** [comment]
I agree it basically is the same. Though the title is not as explicit in #54. So rather than closing this one as duplicate I suggest to leave it open but at the same milestone.

**Comment by xogeny** [comment]
My current proposal is to handle this via #97.  The idea would basically be that you could specify that a given library is a "development" library and, therefore, should be checked out.
